src/WingetCreateCLI/Commands/NewCommand.cs

```diff
@@ -116,9 +116,17 @@ public override async Task<bool> Execute()
                     PackageParser.ParsePackages(packageFiles, this.InstallerUrls, manifests, out List<PackageParser.DetectedArch> detectedArchs);
                     DisplayMismatchedArchitectures(detectedArchs);
                 }
-                catch (ParsePackageException exception)
+                catch (Exception e)
                 {
-                    exception.ParseFailedInstallerUrls.ForEach(i => Logger.ErrorLocalized(nameof(Resources.PackageParsing_Error), i));
+                    if (e is ParsePackageException exception)
+                    {
+                        exception.ParseFailedInstallerUrls.ForEach(i => Logger.ErrorLocalized(nameof(Resources.PackageParsing_Error), i));
+                    }
+                    else if (e is IOException)

```

Comment:
There could be other reasons we hit an IOException, it's a pretty broad exception: [https://docs.microsoft.com/en-us/dotnet/standard/io/handling-io-errors](https://docs.microsoft.com/en-us/dotnet/standard/io/handling-io-errors)

Is there a specific property (HResult, message, innerexception, etc) on the exception that you can use to narrow down to this scenario?

Reply:
Added an additional if statement that checks the HRESULT of the exception to determine if its a virus exception.

Reply:
you can use an exception filter to avoid the else clause: [https://docs.microsoft.com/en-us/dotnet/standard/exceptions/using-user-filtered-exception-handlers](https://docs.microsoft.com/en-us/dotnet/standard/exceptions/using-user-filtered-exception-handlers)

Reply:
Added exception filter